learn/avalanche-bridge-faq.md

```diff
@@ -0,0 +1,177 @@
+The transactions that correspond to bridge transfers will appear on explorers for the Avalanche and Ethereum networks. It may take a few minutes for the transactions to appear. To search for your transaction in the explorer, copy and paste your address into [Avalanche's C-Chain Explorer](https://cchain.explorer.avax.network/) or [Etherscan](https://etherscan.io/). To view the transactions sent by the bridge itself, you can look [here](https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04/transactions) for Avalanche and [here](https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0) for Ethereum. If you still don't see your transaction, reach out on [Telegram](https://t.me/avalancheavax) or [Discord](https://chat.avax.network/). 
+
+### Are there tutorials on how to use the bridge?
+Yes, you can view video tutorials for bridge functionality [here](https://www.youtube.com/playlist?list=PLRHl-ulWK4-FPRA7SS1OrCOC8cOc2K8sP).
+
+### Can I send to a different address on the other network?
+The bridge only allows transfers to the same address on the other network. 
+After the asset is transferred to the other network, it can be sent to any address or contract.
+ 
+### Can I speed up my transaction? <a id="speed-up-transaction"></a>
+Yes, you can click on the “Speed Up” button on Metamask. 
+“Speeding up” a transaction through Metamask issues a new transaction on Ethereum that has a higher gas price than the transaction that was originally sent.
+Since the new transaction has a higher gas price, it is more likely to be included in a block.
+Only one of the transactions (the original and the "sped up") will be accepted.
+Speeding up a transaction that is transferring funds to the bridge is safe.
+However, the user interface will not be aware of the new transaction, meaning you may not see the confirmations in the user interface. 
+Once the new transaction has 35 confirmations on Ethereum, check your Metamask wallet on Avalanche to see the wrapped funds.
+
+### Why does the number of tokens shown on Metamask not match the number I specified?
+When transferring from Avalanche to Ethereum, Metamask shows that 0 tokens are to be transferred, not the actual number of tokens. This is a known problem with Metamask.
+
+## Fees
+### How do fees work on the Avalanche Bridge?
+The bridge charges transfer fees in order to cover the cost of the transaction fees on the Avalanche and Ethereum networks.
+These fees are charged in-kind with the ERC20 asset being transferred.
+That is, when you transfer a token, a portion of the balance transferred is taken by the AB as a fee.
+
+The fee is a percentage of the estimated Ethereum transaction fee, plus a constant amount (currently $5) to account for price volatility.
+
+For a limited time, this percentage is zero to encourage users to transfer assets.
+Further, transfers to Avalanche may qualify for an AVAX airdrop as described [here](#airdrop).
+
```

Comment:
@learyce @jpop32 can you review answer?

Reply:
Conor Leary  1 minute ago
Looks good, I've been messaging with Dan directly to re-write some of the answers